### PR TITLE
Bookstore: Initialize project with components

### PR DESCRIPTION
**Pull request by MohammedElseddik**
## In this branch I:

- Added Navbar component.
- Added Books list component.
- Added Book component.
- Added form input component to add a new book to the list.
- Added Categories component.

## Note:
- The styles will be added in the next milestones.
- I used dummy values of books just to get the UI for this stage of the project, dynamic values will be added in the next project milestone.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import Books from './components/Books/Books';
+import Categories from './components/Categories/Categories';
+
+function App() {
+  return (
+    <BrowserRouter>
+      <NavBar />
+      <main>
+        <Routes>
+          <Route path="/books" element={<Books />} />
```
Comment (EricMbouwe) [inline src/App.js]: - [ ] In order to not show a blank page when visiting the site for the first time, the BookList component should be the default page. So let's add the `exact` prop to the route and change the `path` value to `"/"` to handle that.

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @MohammedElseddik , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
- Linters ok
- Compoments correctly setup
- PR ok

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (MohammedElseddik) [reply src/App.js]: @EricMbouwe Thank you so much for your review but 
could you explain more what you mean by "So let's add the exact prop to the route" ?

Comment (EricMbouwe) [reply src/App.js]: `exact` is a route prop that create the exact matching between the url route entered by the user, the path you defined in your route tag and the component you want your app to render when that url is called.

[more info about](https://www.oreilly.com/library/view/react-router-quick/9781789532555/13c3acc3-c0a2-48fc-a0a6-e3a780e6cad7.xhtml)

**Review by AshakaE — APPROVED** [review]
Hi @MohammedElseddik  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**